app/modules/asset_groups/models.py

```diff
@@ -87,27 +88,70 @@ def get_elasticsearch_schema(cls):
 
     def __init__(self, asset_group, sighting_config, detection_configs):
         self.asset_group = asset_group
-        self.config = sighting_config
+        self.config = {
+            'sighting': sighting_config,
+            'detections': detection_configs,
```

Comment:
I have to admit it's not clear to me why this has been separated out in the changes here BUT I think it's the right thing to do as I expect that very soon the number of 'blocked'/'stuck' AGS will become a higher priority issue and therefore the ability to rerun the detection at a later date drived by a REST API will become required.